tests/Behat/DrupalContext.php

```diff
@@ -201,4 +202,64 @@ protected function getLanguageIdByName(string $name): string {
     throw new \InvalidArgumentException("Language '{$name}' not found.");
   }
 
+  /**
+   * Redirect user on Node creation page.
+   *
+   * @param string $content_type_name
+   *   Content type name.
+   *
+   * @Given I am visiting the :content_type_name create node page
+   */
+  public function iAmVisitingTheCreateNodePage(string $content_type_name): void {
```

Comment:
Not sure we need this custom step. We could simply do "Given I am in the "node/add/oe_demo_translatable_page". The path is quite straightforward and easy to read, but that is my opinion. If we decide to keep the custom step I think it needs to be rewritten to something like "Given I am visiting the :content_type_name creation page"

Reply:
Done